test/controllers/pages_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class PagesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```

Comment:
I'm guessing you will add tests for this in the next lesson. My apologies I do not have time to go through the lessons so I review based off of the code you've written.